rinja/src/lib.rs

```diff
@@ -83,6 +84,7 @@ pub use rinja_derive::Template;
 pub use crate as shared;
 pub use crate::error::{Error, Result};
 pub use crate::helpers::PrimitiveType;
+pub use crate::values::{NO_VALUES, Value, Values};
```

Comment:
I would also reexport `get_value` since people can always call `rinja::get_value` in their templates or even in their code (who knows?).